### DISCUSSION TITLE
I cannot get -trim to work

### DISCUSSION THREAD

**Discussion by ewandavis** [ImageMagick/ImageMagick#2179] [Help]
I can't get trim to work. I have a large number of images with surrounded by unwanted clean white border.

I have tried.

convert cropped.png -fuzz 10% -trim -alpha off +repage result2.png
with and without -alpha off and with various values for fuzz. The white border looks clean in GIMP

The image output looks unchanged  and still has it's white border. Same issue with jpeg.

I attach a sample image.

Any help would be much appreciated

Thanks

Ewan

![cropped](https://user-images.githubusercontent.com/16081527/85207820-83854d00-b323-11ea-82d3-c3b48c88852a.png)

**Comment by fmw42** [comment]
You have two issues. First, you have a dark border. Second you have some dark spots. So you can just shave off a few pixels on the sides and then increase the fuzz value. So this works for me

`convert picture.png -shave 5x5 -fuzz 50% -trim +repage result.png`

![result](https://user-images.githubusercontent.com/13458218/85208060-4482d780-b2e2-11ea-805a-30b3c8a17e51.png)

**Comment by ewandavis** [comment]
Thanks, that works
